server.go

```diff
@@ -770,9 +780,22 @@ func (s *Server) syncLocalOrderState() error {
 			context.Background(), orderNonce,
 		)
 		if err != nil {
-			return fmt.Errorf("unable to fetch order(%v): %v",
-				orderNonce, err)
+			switch {
+			// If the server does not know about this order mark it
+			// as failed and allow the client start up with the
+			// rest of valid orders.
+			//
+			// NOTE: This should never happen.
+			case clientdb.IsOrderNotFoundErr(err):
```

Comment:
Yeah, reading this I first thought this meant "the local DB didn't find the order". So moving the function to the auctioneer package should make that more clear.